website/templates/view-campaign.html

```diff
@@ -69,30 +66,21 @@
             {% endfor %}
           </span>
         </li>
-
-        <li role="separator" class="mdc-list-divider"></li>
-        <li class="mdc-list-item">
-          <span class="mdc-list-item__text">Closes</span>
```

Comment:
Do we mean to not show users when a campaign closes?

(We can comment this out **and/or** add a TODO issue in GitHub if we don't.)

Reply:
Removed since this is stubbed information, data we have doesn't provide closing date and the form we have to create a new campaign doesn't allow users to input closing date.